libraries/AP_HAL_ChibiOS/hwdef/scripts/dma_resolver.py

```diff
@@ -340,7 +340,24 @@ def write_dma_header(f, peripheral_list, mcu_type, dma_exclude=[],
     for p in peripheral_list:
         forbidden_map[p] = forbidden_list(p, peripheral_list)
 
+    # force sharing of TIMx_UP and TIMx_CHy if possible
+    periphs = peripheral_list.copy()
```

Comment:
need configure_all.py plus dirdiff to see the impact of this

Reply:
Done - no changes